taskt/Core/Automation/Commands/IE Browser Commands/IECreateBrowserCommand.cs

```diff
@@ -1,61 +1,75 @@
-﻿using System;
+﻿using SHDocVw;
+using System;
 using System.Collections.Generic;
+using System.Threading;
 using System.Windows.Forms;
 using System.Xml.Serialization;
-using taskt.UI.CustomControls;
-using taskt.UI.Forms;
 using taskt.Core.App;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
 using taskt.Core.Utilities.CommonUtilities;
+using taskt.UI.CustomControls;
+using taskt.UI.Forms;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("IE Browser Commands")]
-    [Attributes.ClassAttributes.Description("This command allows you to create a new IE web browser session.")]
-    [Attributes.ClassAttributes.ImplementationDescription("This command implements the 'InternetExplorer' application object from SHDocVw.dll to achieve automation.")]
+    [Group("IE Browser Commands")]
+    [Description("This command creates a new IE Web Browser Session.")]
     public class IECreateBrowserCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please Enter the instance name")]
+        [PropertyDescription("Instance Name")]
+        [InputSpecification("Specify a name or a variable for a new IE Browser Instance.")]
+        [SampleUsage("IEBrowser || {vIEBrowser}")]
+        [Remarks("")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
```

Comment:
There's no variable intake for creating an instance. So we can remove the helper, the {vIEBrowser} sample, and the "variable" in input specification. This is the input specification I have in excel and word for creating. 

`[InputSpecification("Enter a unique name that will represent the application instance. " +
            "This unique name allows you to refer to the instance by name in future commands, " +
            "ensuring that the commands you specify run against the correct application.")]`

Reply:
done